includes/wikia/GlobalTitle.php

```diff
@@ -835,6 +844,32 @@ private function loadNamespaceNames() {
 	}
 
 	/**
+	 * Get $wgExtraNamespaces for title language
+	 *
+	 * @return array
+	 */
+	private function getLocalWgExtraNamespaces() {
+		global $wgExtensionNamespacesFiles;
+		$extraNamespaces = [ ];
+
+		foreach ( $wgExtensionNamespacesFiles as $extFile ) {
+			$namespaces = [ ];
+			// Namespace files fill in $namespaces array
+			require( $extFile );
+
+			foreach ( $namespaces as $langKey => $namespaceArray ) {
```

Comment:
considering this and https://github.com/Wikia/app/pull/11501/files#r81142282 I suspect it'd be better to have all langs in a static variable and reuse it in other global title. This way we optimize number of hard disk reads operation by sacrificing few additional iterations.